Dockerfile

```diff
@@ -25,26 +25,30 @@ COPY --from=frontend-builder /app/.output/public ./app/api/static/public
 RUN CGO_ENABLED=0 GOOS=linux go build \
     -ldflags "-s -w -X main.commit=$COMMIT -X main.buildTime=$BUILD_TIME -X main.version=$VERSION"  \
     -o /go/bin/api \
-    -v ./app/api/*.go
+    -v ./app/api/*.go && \
+    chmod +x /go/bin/api && \
+    # create a directory so that we can copy it in the next stage
+    mkdir /data
 
 # Production Stage
-FROM alpine:latest
+FROM gcr.io/distroless/static

```

Comment:
Any reason to use this over `FROM scratch` ?

Reply:
From my perspective I think we do want a user, we do need CA-certificates in case we want to make some TLS connection outside (e.g., email server) and we want to retain timezone data. 

`static` distroless is `scratch` + user, timezone and CA-certs, so essentially the same image we would build ourselves if we were to then add all those manually FROM `scratch`. 

As much as I am not thrilled to use Google's products, I think in this case `distroless` is a better fit compared to `scratch` to keep the Dockerfile cleaner.